rust-roolchain

```diff
@@ -0,0 +1 @@
+1.22.1
```

Comment:
do we need this?

Reply:
I don't think so, the `cargo fmt` check was using that version, so I put it in.